Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -14,17 +14,31 @@ public protocol DiskMemoryPersistenceStackDelegate: class {
 
 public final class DiskMemoryPersistenceStack: NSObject, PersistenceStack {
 
-    private enum Constants {
-
-        enum PerformanceMetrics {
-            static let totalMemoryKey = "total_memory"
-            static let totalDiskKey = "total_disk"
-
-            static let writeMemoryKey = "write_memory"
-            static let readMemoryKey = "read_memory"
-
-            static let writeDiskKey = "write_disk"
-            static let readDiskKey = "read_disk"
+    public struct PersistencePerformanceMetrics {
+
+        let metrics: PerformanceMetrics
+        let memoryAttributeKey: String
+        let diskAttributeKey: String
+        let readMemoryEventKey: String
+        let writeMemoryEventKey: String
+        let readDiskEventKey: String
+        let writeDiskEventKey: String
+
+        public init(metrics: PerformanceMetrics,
+                    memoryAttributeKey: String,
```

Comment:
we could use the previous constant values as default values 💅

Reply:
Fixed